examples/models/cpp/simple-cpp/README.md

```diff
@@ -0,0 +1,157 @@
+# Single CPP File Build
+
+In this example we will show how we can wrap a simple CPP project with the default parameters, which will enable us to just provide a single CPP file.
+
+If you want to wrap an existing library, or you have more complex requirements please refer to the ["Build system override CPP Wrapper example"]().
+
+You can read about how to configure your environment in the [CPP Wrapper page]().
```

Comment:
nit: same here, missing link to the wrapper root page

Reply:
Good shout!

Reply:
Added link